pkgs/os-specific/linux/kernel/xanmod-kernels.nix

```diff
@@ -44,9 +44,9 @@ let
       # WineSync driver for fast kernel-backed Wine
       WINESYNC = module;
 
-      # Preemptive Full Tickless Kernel at 500Hz
-      HZ = freeform "500";
-      HZ_500 = yes;
+      # Preemptive Full Tickless Kernel at 250Hz
+      HZ = freeform "250";
+      HZ_250 = yes;
```

Comment:
Could you add a note to the commit message why this was done?

Reply:
added to the commit message